apps/judicial-system/web/messages/Core/index.ts

```diff
@@ -59,4 +59,24 @@ export const core = defineMessages({
     description:
       'Notað fyrir texta á hnappi til að sækja vistunarseðil sem pdf skjal.',
   },
+  caseNumber: {
+    id: 'judicial.system.core:case_number',
+    defaultMessage: 'Mál nr.',
```

Comment:
Please consider using 

`defaultMessage: 'Mál nr. {caseNumber}'`

Then when using this string you can do

```
formatMessage(m.caseNumber, {
    caseNumber: workingCase.caseNumber
})
```